docs/reference/src/documentation/introduction/installation/source.md

```diff
@@ -4,7 +4,7 @@ The `Sway toolchain` can be built directly from the [`Sway repository`](https://
 
 ## Installation & Updating
 
-In the root of the repository `/sway/<here>` build [`forc`](../../forc/commands/index.md) with the following command:
+In the root of the repository `/sway/<here>` build [`forc`](https://fuellabs.github.io/sway/master/book/forc/commands/index.html) with the following command:
```

Comment:
Is it better to use the relative path, in case we change the DNS at some point?

Reply:
It doesn't seem like the relative paths work across books

Reply:
Just a point to consider: 

Does reference has versions? i feel like linking like this would make versioned books linking to latest always. Example:

Imagine we released 0.43 with this merged. And after the merge we change some stuff in the place this link points. If a user goes to 0.43's documentation they should be taken to that link as it is while we were merging.

Reply:
that's a good point - if I add a version number here though will someone update it for each release?

Reply:
That is a good question :) It is highly possible that we will forget that, we can try to do that until we can find a way to link relatively to a different book